convert/convert_1_0_to_1_1.py

```diff
@@ -0,0 +1,72 @@
+from jsonschema import validate
+import json
+
+def convert_1_0_to_1_1(old):
+    new = {
+        "asOfDateTime": old.get("asOfDateTime", ""),
+        "columns": [],
+        "creationDateTime": old.get("creationDateTime", ""),
+        "datasetJSONVersion": old.get("datasetJSONVersion", ""),
+        "fileOID": old.get("fileOID", ""),
+        "label": old.get("fileOID", ""),
+        "metaDataRef": old.get("metaDataRef", ""),
+        "metaDataVersionOID": old.get("metaDataVersionOID", ""),
+        "name": old.get("fileOID", ""), 
+        "originator": old.get("originator", ""),
+        "records": 0,
+        "sourceSystem": old.get("sourceSystem", ""),
+        "sourceSystemVersion": old.get("sourceSystemVersion", ""),
+        "studyOID": old.get("studyOID", ""),
```

Comment:
`studyOID` is in `clinicalData` in V1.0 I think

Reply:
Good spots!